apps/backend/prisma/schema.prisma

```diff
@@ -135,3 +137,11 @@ model PairConnection {
   connectionScore Decimal?
   createdAt       DateTime @default(now())
 }
+
+model TapGraphEdge {
+    id          String @id @default(uuid())
+    tapperId      String? // Hash(Hash(Private Signing Key)) : use double hash so that single hash can be used for revocation
```

Comment:
Let's use more clear terminology here, so one letter typos do not create bugs in the future - perhaps replace tapperId with tapSenderId and tappedId with tapReceiverId? this might be confusing as well, but we should come up with something else here

Reply:
Reasonable. tapSender and tapReceiver is definitely better. Will keep thinking.

Reply:
Addressed.